gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
```

Comment:
```distributionUrl=https\://services.gradle.org/distributions/gradle-7.0-bin.zip```

Kyle, what do you think about Gradle 7.0? The [release notes](https://docs.gradle.org/7.0/release-notes.html) look attractive. I am personally still on 6.8.3 on my machine and am already seeing deprecations that will break in 7.0.

Reply:
I'm cool with whatever version, as long as Spring supports it. I just used whatever https://start.spring.io spit out.